pip-proto/service-gen.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
As mentioned earlier this should be replaced with `go:genereate` comments in related packages.